R/geom-contour.r

```diff
@@ -68,5 +68,10 @@ geom_contour <- function(mapping = NULL, data = NULL,
 #' @include geom-path.r
 GeomContour <- gganimintproto("GeomContour", GeomPath,
   default_aes = aes(weight = 1, colour = "#3366FF", size = 0.5, linetype = 1,
-    alpha = NA)
+    alpha = NA),
+    pre_process = function(g, g.data, ...) {
```

Comment:
can you please standardize the indentation? remove spaces before pre_process

Reply:
Just give me a few minutes.

Reply:
I've updated this.